products/caregivers/10-10CG Form/Existing Monitors and Alerts.md

```diff
@@ -1,51 +1,83 @@
 # Monitors and Alerts for the 10-10CG application
-- Last Edit date: 10/16/2023
+
+- Last Edit date: 11/15/2024
 
 This document lists the various monitors and alerts in place for the 10-10CG.
 
 All alerts are set up to populate the **#health-tools-1010-apm** slack channel.
 
-
 ## Monitor: 10-10CG anomaly monitor
+
 ### Triggered: 1010CG anomaly monitor
+
 #### Alert Reason:
-- There have been less than 25 submissions in the last 12 hours
+
+- There have been fewer than 25 submissions in the last 12 hours
+
 #### Action:
+
 - 10-10 Health Apps team will determine if there have been any failures that is causing the low volume of submissions
 - If any errors/failures are found, this information will be communicated on the **#vecms-carma_vadotgov_development_external** Slack channel
+
 ### Recovered: 1010CG anomaly monitor
+
 #### Alert Reason
+
 - The volume of submissions has reached at or above 25 submissions over the last 12 hours
+
 #### Action:
+
 - No Action needed, this alert is letting us know that volumes are back to normal
+
 ## Monitor: 1010CG submission job has failed retries
+
 ### Triggered: 1010CG submission job has failed ten retries
+
 #### Alert Reason
-- At least one application submission has been retried 10 times, and has failed.  The submission will retry 4 more times before failing completely.
+
+- At least one application submission has been retried 10 times, and has failed. The submission will retry 6 more times before failing completely.
```

Comment:
This was changed from 14 to 16 for most sidekiq jobs to allow for handling COLA outages. https://github.com/department-of-veterans-affairs/vets-api/pull/19289 for the PR.